x-pack/solutions/security/plugins/security_solution/common/siem_migrations/constants.ts

```diff
@@ -8,6 +8,8 @@
 export const SIEM_MIGRATIONS_ASSISTANT_USER = 'assistant';
 
 export const SIEM_MIGRATIONS_PATH = '/internal/siem_migrations' as const;
+
+// TODO: Move `SIEM_RULE_MIGRATIONS_PATH` and composed paths to rules/constants.ts
 export const SIEM_RULE_MIGRATIONS_PATH = `${SIEM_MIGRATIONS_PATH}/rules` as const;
```

Comment:
Will be done in separate PR. A sub folder of `rules/` will be good but will unncessary increases the number of files in this PR.